index.html

```diff
@@ -4,37 +4,170 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio Site</title>
+  <title>Portfolio</title>
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
-  
 </head>
 <body>
-  <header class="head">
-    <nav>
-      <a href="#">Portfolio</a>
-      <i class="fa fa-bars"></i>   
-    </nav>
-  </header>
-  <main>
-    <section class="container">
+    <header class="hero">
+      <nav>
+        <a href="#">Portfolio</a>
+        <i class="fa fa-bars"></i>
+      </nav>
+    <main class="container">
       <h1>
-      I'm Uchechi<br>
-      Glad to see<br>
-      you!
-      </h1>
-      <p>
-      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <h3>LETS CONNECT</h3>
-      <ul>
-        <li><a href="#"><img src="images/Tweeter.png"></a></li>
-        <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
-        <li><a href="#"><img src="images/m.png"></a></li>
-        <li><a href="#"><img src="images/Github.png"></a></li>
-        <li><a href="#"><img src="images/tech.png"></a></li>
+        I'm Uchechi<br>
+        Glad to see<br>
+        you!
+        </h1>
+        <p>
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h3>LETS CONNECT</h3>
+        <ul #="social">
+          <li><a href="#"><img src="images/Tweeter.png"></a></li>
+          <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
+          <li><a href="#"><img src="images/m.png"></a></li>
+          <li><a href="#"><img src="images/Github.png"></a></li>
+          <li><a href="#"><img src="images/tech.png"></a></li>
+        </ul>
```

Comment:
- [ Optional] Kindly fix the position of your social media icons as per the Figma design.
 
![rev2](https://user-images.githubusercontent.com/53347966/198839236-c51faf39-92ee-4fb2-8771-86ef534eaa10.JPG)